a/aether/aether-api-n-spi-ubi8.4.sh

```diff
@@ -0,0 +1,55 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : aether-api, aether-spi
+# Version       : aether-1.13.1
+# Source repo   : https://github.com/sonatype/sonatype-aether
+# Tested on     : UBI: 8.4
+# Script License: Apache License 2.0
+# Maintainer's  : Sapana Khemkar <Sapana.Khemkar@ibm.com>
```

Comment:
From now onwards, please start adding a new field `Language` to the script header. This is applicable for all the build scripts.
E.g:
```
# Maintainer's : Sapana Khemkar <Sapana.Khemkar@ibm.com>
# Language : Java
```

Reply:
Noted. Changes done for this PR